helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -35,10 +35,10 @@ vertical-pod-autoscaler:
 
     resources:
       limits:
-        cpu: 50m
+        cpu: 100m
```

Comment:
Here, you keep the requests/limits ratio 1:1. Is there a reason not to leave it some "wiggle room", like the others?

Reply:
Even in large clusters the admission controller does not use more than 50millicores.  
Having said that, I agree it's best to give it more room